client/blocks/reader-excerpt/index.jsx

```diff
@@ -37,9 +40,49 @@ const convertExcerptNewlinesToBreaks = ( excerpt ) => {
 	return excerpt;
 };
 
+/**
+ * Removes double spaces and also &nbsp; characters which may be present in the excerpt.
+ *
+ * @param {string} str the string to normalize
+ * @returns a string with single space characters.
+ */
+const normalizeWhitespace = ( str ) => {
+	return str.replace( /\s+/g, ' ' );
+};
+
+/**
+ * Gets the writing prompt text which was inserted as a pullquote at the begining of the post's content.
+ *
+ * @param {Object} post the post object
+ * @returns writing prompt text
+ */
+const getDailyPromptText = ( post ) => {
+	const dom = domForHtml( post.content );
+	const promptQuote = dom.querySelector( '.wp-block-pullquote:first-child' );
```

Comment:
```suggestion
	const promptQuote = dom.querySelector( '.wp-block-pullquote:first-of-type' );
```
I found this more reliable when parsing out the prompt quote

Reply:
actually, I do want to assert that it's the first block in the post content as well, because someone may have deleted the writing prompt blockquote but have one of their own unrelated blockquotes embedded in the post